apps/ccd/ithc/base/kustomization.yaml

```diff
@@ -9,6 +9,7 @@ resources:
   - ../../message-publisher/message-publisher.yaml
   - ../../ccd-case-disposer/ccd-case-disposer.yaml
   - ../../../base/slack-provider/ithc
+
```

Comment:
```suggestion
- ../../ccd-logstash-civil
```

Reply:
You usually add this here: https://github.com/hmcts/cnp-flux-config/blob/e58b48f8f364990cbbceaa531f18e833c85df617/apps/ccd/logstash/kustomization.yaml but because you're only adding it to ithc now you need to put it at a different level

Reply:
We would like test only on ithc env at the moment, so we did not make any change in main lagstash /cnp-flux-config/apps/ccd/logstash,  If we change generic settings, it may impact other env as per Dan Lysiak suggestions. Would you be able to suggested change so we can make change ?